Core/GDCore/String.cpp

```diff
@@ -10,7 +10,6 @@
 
 #include <algorithm>
 #include <string.h>
-
 #include "GDCore/CommonTools.h"
```

Comment:
This file only have empty line changes. It should be reverted to have a clearer history.